src/main/java/org/bytedeco/javacpp/tools/Parser.java

```diff
@@ -3166,7 +3185,7 @@ boolean typedef(Context context, DeclarationList declList) throws ParserExceptio
                         info.valueTypes(info.pointerTypes != null ? info.pointerTypes : new String[] {typeName});
                         info.pointerTypes("PointerPointer");
                     } else if (info.pointerTypes == null) {
-                        info.pointerTypes(typeName);
```

Comment:
A fix not directly related to this PR.
`typeName` is a cpp name. We need a java name for `pointerTypes`. Or did I miss something ?

Reply:
I let this change in. 
I think it fixes a bug but I can revert the change, or move it to a 1-line PR if you want.

Reply:
Yes, your fix should be fine, thanks.